index.html

```diff
@@ -0,0 +1,242 @@
+          </li>
+        </ul>
+        <!-- </div> -->
+
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+
+        <div class="list-container-2">
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <button type="button" class="big-button">See project</button>
+      </div>
+      <!---work section card 4-->
+      <div class="workd-card">
+        <img src="images/SnapshootPortfolio4.png" alt="" />
+        <h1>Multi-Post Stories</h1>
+
+        <ul class="list-container-1">
+          <li class="canopy">canopy</li>
+
+          <li>
+            <img src="images/Counter.png" alt="" />
+            Back End Dev
+          </li>
+
+          <li>
+            <img src="images/Counter.png" alt="" />
+            2015
+          </li>
+        </ul>
+        <!-- </div> -->
+
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+
+        <div class="list-container-2">
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <button type="button" class="big-button">See project</button>
+      </div>
+      <!---about my self-->
+      <div class="workd-card about-myself">
+        <h1>About Myself</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <p id="text-color">LET’S CONNECT</p>
+
+        <ul>
+          <li><i class="fab fa-twitter"></i></li>
+        <li><i class="fab fa-linkedin"></i></li>
+        <li><i class="fab fa-medium"></i></li>
+        <li><i class="fab fa-github"></i></li>
+        <li><i class="fab fa-angellist"></i></li>
+      
+        </ul>
+
+        <button type="button" class="big-button below-button">
+          Get my resume
+        </button>
+      </div>
+      <!---about my self end-->
+
+      <div class="frames">
+        <div class="down-arrow space">
+          <h4>
+            Languages
+            <i class="fa fa-arrow-down"></i>
+          </h4>
+        </div>
+
+        <img src="images/Frame9.png" alt="" />
+        <img src="images/Frame10.png" alt="" />
+        <img src="images/Frame11.png" alt="" />
+
+        <img src="images/Vector.png" alt="" />
+        <h4 class="down-arrow">
+          Frameworks
+
+          <i class="fa fa-arrow-right"></i>
+        </h4>
+        <img src="images/SeparatorBottom.png" alt="" />
+
+        <h4 class="down-arrow">
+          Skills
+          <i class="fa fa-arrow-right"></i>
+        </h4>
+        <img src="images/SeparatorBottom.png" alt="" />
+      </div>
```

Comment:
- [ ] All the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. Kindly look into this and refactor your code.